app/assets/stylesheets/modules/_family.scss

```diff
@@ -29,10 +29,12 @@
 
       background: image-url("julian-michelle.png");
       background-repeat: no-repeat;
-      background-size: 100%;
+      background-size: em(282);
+      width: em(282);
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height, margin, margin-bottom, width